Update tools chapter (#315)
pmix/pmix-standard: Chap_API_Tools.tex

        \item If \refattr{PMIX_TOOL_RANK} is also given, then the rank of the tool will be assigned that value.
        \item If \refattr{PMIX_TOOL_RANK} is not given, then the rank will be set to a default value of zero.
    \end{itemize}

>     \item If a process ID has not been provided and the tool connects to a server, then one will be assigned by the host environment upon connection to that server.

Reviewer: ```suggestion
    \item If a process ID is not provided and the tool connects to a server, then one will be assigned by the host environment upon connection to that server.
```
Just to match the wording of the point that follows.
